janusgraph-core/src/main/java/org/janusgraph/core/schema/JanusGraphManagement.java

```diff
@@ -401,4 +401,43 @@
      */
     public void rollback();
 
+    /*
+    ##################### PRINT SCHEMA ELEMENTS ##########################
+     */
+
+    /**
+     * Prints out schema infomation related to vertex and edge labels, indexes, and property keys.
+     *
+     * @return a collection of all the mgmt API schema printing methods
+     */
+    public String printSchema();
+
+    /**
+     * Prints out schema infomation related to vertex labels.
+     *
+     * @return String with vertex label schema information
+     */
+    public String printVertexLabels();
+
+    /**
+     * Prints out schema infomation related to edge labels.
+     *
+     * @return String with edge label schema information
+     */
+    public String printEdgeLabels();
+
+    /**
+     * Prints out schema infomation related to property keys.
+     *
+     * @return String with property key schema information
+     */
+    public String printPropertyKeys();
+
+    /**
+     * Prints out schema infomation related to Composite, Mixed, Vertex, Edge, and relation indexes
```

Comment:
(nitpick) I would not enumerate all kinds of indexes JanusGraph supports here and instead just say that it returns information about index as we probably won't remember that we need to update this comment in case we add another type of index or remove one.